### PR TITLE
Rename the method name

### PR DESCRIPTION
**Pull request by matthias-ronge**

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/services/image/ImageGenerator.java [hunk 1]
```diff
@@ -311,12 +311,12 @@ public GenerationMode getMode() {
      * Lets the supervisor do something, if there is one. Otherwise nothing
      * happens.
      *
-     * @param action
+     * @param something
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/services/image/ImageGenerator.java]: If it is an action, call it action, not "something"

Comment (Erikmitk) [reply Kitodo/src/main/java/org/kitodo/services/image/ImageGenerator.java]: @matthias-ronge Have you seen this comment? I agree with @Kathrin-Huber that you should change the comment and not the parameter.

`Let's the supervisor perform an action, if there is one.` is more fitting for the method.

Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/services/image/ImageGenerator.java]: Okay, I misunderstood! I thought the method signature was meant. I had not looked at the Javadoc.

**Comment by Kathrin-Huber** [comment]
Please resolve colflicts

**Pull request merged**